tests/Behat/Page/Admin/OrderRefundsPage.php

```diff
@@ -121,11 +121,7 @@ public function isPaymentMethodSelected(string $paymentMethodName): bool
             return false;
         }
 
-        if ($optionField->getText() != $paymentMethodName) {
```

Comment:
Could be part of separate PR so that we would merge it already :)

Reply:
Yeah, that commit was actually just sitting in my local fork and got added to this PR. I will create another PR for that only. :)

Reply:
For some reason this was already commited. Not sure why github thinks its still a change.

https://github.com/Sylius/RefundPlugin/commit/1d2d0bf9c7fdadc94afddc6783d649d0e5086312#diff-bccc45a8cb97fcb3c44b347a0a795920